ports/nrf/boards/feather52/pins.c

```diff
@@ -0,0 +1,124 @@
+
+const pin_obj_t pin_PA19 = PIN(A, 19, NULL, NO_ADC);
+const pin_obj_t pin_PA20 = PIN(A, 20, NULL, NO_ADC);
+
+const pin_obj_t pin_PA25 = PIN(A, 25, NULL, NO_ADC);
+const pin_obj_t pin_PA26 = PIN(A, 26, NULL, NO_ADC);
+const pin_obj_t pin_PA27 = PIN(A, 27, NULL, NO_ADC);
+const pin_obj_t pin_PA28 = PIN(A, 28, NULL, SAADC_CH_PSELP_PSELP_AnalogInput4);
+const pin_obj_t pin_PA29 = PIN(A, 29, NULL, SAADC_CH_PSELP_PSELP_AnalogInput5);
+const pin_obj_t pin_PA30 = PIN(A, 30, NULL, SAADC_CH_PSELP_PSELP_AnalogInput6);
+const pin_obj_t pin_PA31 = PIN(A, 31, NULL, SAADC_CH_PSELP_PSELP_AnalogInput7);
+
+STATIC const mp_rom_map_elem_t mcu_pin_globals_table[] = {
+  { MP_ROM_QSTR(MP_QSTR_PA02), MP_ROM_PTR(&pin_PA02) },
+  { MP_ROM_QSTR(MP_QSTR_PA03), MP_ROM_PTR(&pin_PA03) },
+  { MP_ROM_QSTR(MP_QSTR_PA04), MP_ROM_PTR(&pin_PA04) },
+  { MP_ROM_QSTR(MP_QSTR_PA05), MP_ROM_PTR(&pin_PA05) },
+  { MP_ROM_QSTR(MP_QSTR_PA06), MP_ROM_PTR(&pin_PA06) },
+  { MP_ROM_QSTR(MP_QSTR_PA07), MP_ROM_PTR(&pin_PA07) },
+  { MP_ROM_QSTR(MP_QSTR_PA08), MP_ROM_PTR(&pin_PA08) },
+  { MP_ROM_QSTR(MP_QSTR_PA09), MP_ROM_PTR(&pin_PA09) },
+  { MP_ROM_QSTR(MP_QSTR_PA10), MP_ROM_PTR(&pin_PA10) },
+  { MP_ROM_QSTR(MP_QSTR_PA11), MP_ROM_PTR(&pin_PA11) },
+  { MP_ROM_QSTR(MP_QSTR_PA12), MP_ROM_PTR(&pin_PA12) },
+  { MP_ROM_QSTR(MP_QSTR_PA13), MP_ROM_PTR(&pin_PA13) },
+  { MP_ROM_QSTR(MP_QSTR_PA14), MP_ROM_PTR(&pin_PA14) },
+  { MP_ROM_QSTR(MP_QSTR_PA15), MP_ROM_PTR(&pin_PA15) },
+  { MP_ROM_QSTR(MP_QSTR_PA16), MP_ROM_PTR(&pin_PA16) },
+  { MP_ROM_QSTR(MP_QSTR_PA17), MP_ROM_PTR(&pin_PA17) },
+  { MP_ROM_QSTR(MP_QSTR_PA19), MP_ROM_PTR(&pin_PA19) },
+  { MP_ROM_QSTR(MP_QSTR_PA20), MP_ROM_PTR(&pin_PA20) },
+  { MP_ROM_QSTR(MP_QSTR_PA25), MP_ROM_PTR(&pin_PA25) },
+  { MP_ROM_QSTR(MP_QSTR_PA26), MP_ROM_PTR(&pin_PA26) },
+  { MP_ROM_QSTR(MP_QSTR_PA27), MP_ROM_PTR(&pin_PA27) },
+  { MP_ROM_QSTR(MP_QSTR_PA28), MP_ROM_PTR(&pin_PA28) },
+  { MP_ROM_QSTR(MP_QSTR_PA29), MP_ROM_PTR(&pin_PA29) },
+  { MP_ROM_QSTR(MP_QSTR_PA30), MP_ROM_PTR(&pin_PA30) },
+  { MP_ROM_QSTR(MP_QSTR_PA31), MP_ROM_PTR(&pin_PA31) },
+};
+MP_DEFINE_CONST_DICT(mcu_pin_globals, mcu_pin_globals_table);
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+  { MP_ROM_QSTR(MP_QSTR_A0   ), MP_ROM_PTR(&pin_PA02) },
+  { MP_ROM_QSTR(MP_QSTR_A1   ), MP_ROM_PTR(&pin_PA03) },
+  { MP_ROM_QSTR(MP_QSTR_A2   ), MP_ROM_PTR(&pin_PA04) },
+  { MP_ROM_QSTR(MP_QSTR_A3   ), MP_ROM_PTR(&pin_PA05) },
+  { MP_ROM_QSTR(MP_QSTR_TX   ), MP_ROM_PTR(&pin_PA06) },
+  { MP_ROM_QSTR(MP_QSTR_A7   ), MP_ROM_PTR(&pin_PA07) },
+  { MP_ROM_QSTR(MP_QSTR_RX   ), MP_ROM_PTR(&pin_PA08) },
+  { MP_ROM_QSTR(MP_QSTR_NFC1 ), MP_ROM_PTR(&pin_PA09) },
+  { MP_ROM_QSTR(MP_QSTR_NFC2 ), MP_ROM_PTR(&pin_PA10) },
+  { MP_ROM_QSTR(MP_QSTR_D11  ), MP_ROM_PTR(&pin_PA11) },
+  { MP_ROM_QSTR(MP_QSTR_SCK  ), MP_ROM_PTR(&pin_PA12) },
+  { MP_ROM_QSTR(MP_QSTR_MOSI ), MP_ROM_PTR(&pin_PA13) },
+  { MP_ROM_QSTR(MP_QSTR_MISO ), MP_ROM_PTR(&pin_PA14) },
```

Comment:
Include a D13 here for an LED to be standard with examples. Each internal pin can have multiple names and the first listed will be used when its printed out.